.kokoro/release/publish.sh

```diff
@@ -0,0 +1,2 @@
+# Record the current commit hash.
```

Comment:
The robots (and I) do not like shell scripts without a '#!' line.  We generally use:

```sh
#!/usr/bin/env bash
```

e.g.:

https://github.com/googleapis/google-cloud-cpp/blob/efb80e25aec7706dcbc61c5664ca391990a9a518/ci/kokoro/macos/build.sh#L1